reference/publish-prompt-template.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Publish Prompt Template"
+title: "Publish Prompt Template (Deprecated)"
 openapi: "POST /rest/publish-prompt-template"
 ---
 
```

Comment:
Maybe add a line to this page describing that this worked with the old prompt template schema returned by `promptlayer.prompts.get`

Reply:
Sure. It's updated.